### PR TITLE
Delete reproduce_tool.

### PR DESCRIPTION
**Pull request by oliverchang**
This is deprecated, unmaintained, and unused.

### CODE REVIEW

**Review by Alan32Liu — APPROVED** [review]

File: src/appengine/handlers/testcase_detail/show.py [hunk 1]
```diff
@@ -508,7 +508,7 @@ def get_testcase_detail(testcase):
       'crash_address':
           crash_address,
       'crash_state':
-          crash_state,  # Used by reproduce tool.
+          crash_state,
```
Comment (jonathanmetzman) [inline src/appengine/handlers/testcase_detail/show.py:511]: If reproduce tool was the only use, should we get rid of this. or leave a todo?

File: src/local/butler/bootstrap.py [hunk 2]
```diff
@@ -18,14 +18,10 @@
 from local.butler import common
 
 
-def execute(args):
+def execute(_):
```
Comment (jonathanmetzman) [inline src/local/butler/bootstrap.py:21]: I think you've said that `del X` is the preferred style these days.

**Review by jonathanmetzman — APPROVED** [review]
lgtm

File: src/appengine/handlers/testcase_detail/show.py (hunk 1 above)
Comment (oliverchang) [reply src/appengine/handlers/testcase_detail/show.py:511]: I think this is useful enough to keep.

File: src/local/butler/bootstrap.py (hunk 2 above)
Comment (oliverchang) [reply src/local/butler/bootstrap.py:21]: good call. fixed!

**Comment by josepharhar** [comment]
@oliverchang I used reproduce.sh for reproducing all of my clusterfuzz bugs. Is there an alternative program/command that I can just give a clusterfuzz.com/testcase-detail URL that will reproduce crashes for me?

**Pull request merged**